### PR TITLE
TASK-5888 - REST Error : 414 URI too large and 431 Header too large

### PR DESCRIPTION
**Pull request by j-coll**

### CODE REVIEW

**Comment by halender** [comment]
Task linked: [TASK-5888 REST Error : 414 URI too large and 431 Header too large](https://app.clickup.com/t/36631768/TASK-5888)

File: opencga-core/src/main/java/org/opencb/opencga/core/config/AbstractServerConfiguration.java [hunk 1]
```diff
@@ -49,13 +48,9 @@ public AbstractServerConfiguration setPort(int port) {
...
-    public String getLogFile() {
-        return logFile;
-    }
-
-    public AbstractServerConfiguration setLogFile(String logFile) {
-        this.logFile = logFile;
-        return this;
+    @Deprecated
+    protected void setLogFile(Object o) {
+        Configuration.reportUnusedField("configuration.yml#server.[rest|grpc].logFile", o);
```
Comment (pfurio) [inline opencga-core/src/main/java/org/opencb/opencga/core/config/AbstractServerConfiguration.java:53]: I like this :)

File: opencga-core/src/main/java/org/opencb/opencga/core/config/Configuration.java [hunk 2]
```diff
@@ -200,6 +202,16 @@ private static void overwriteWithEnvironmentVariables(Configuration configuratio
         }
     }
 
+    public static void reportUnusedField(String field, Object value) {
```
Comment (pfurio) [inline opencga-core/src/main/java/org/opencb/opencga/core/config/Configuration.java:205]: I'll definitely be using this method

File: opencga-core/src/test/java/org/opencb/opencga/core/config/ConfigurationTest.java [hunk 3]
```diff
@@ -34,68 +34,7 @@
 public class ConfigurationTest {
 
     @Test
-    public void test() {
```
Comment (pfurio) [inline opencga-core/src/test/java/org/opencb/opencga/core/config/ConfigurationTest.java:37]: I wonder who left this here...

**Review by pfurio — APPROVED** [review]

**Pull request merged**